_src/Evasions/techniques/AVBypass.md

```diff
@@ -0,0 +1,92 @@
+---
+layout: post
+title:  "Evasions: Bypassing Security Solutions"
+title-image: "/assets/icons/AVBypass.svg"
+categories: evasions
+tags: AVBypass
+---
+
+<h1>Contents</h1>
+
+[Bypassing Security Solutions detection methods](#avbypass-detection-methods)
+<br />
+[1. Detect Wine](#detect-wine)
+<br />
+[2. Add to Windows defender exclusion list](#add-to-windows-defender-exclusion-list)
```

Comment:
Do you think this method can be added to the "Registry" chapter? Right now, it describes methods to detect environment via Registry checks, but another paragraph can be dedicated to write interaction with the registry